app/views/shared/_available_languages.html.erb

```diff
@@ -0,0 +1,15 @@
+<% if non_english_translations(translations).any? %>
```

Comment:
This is the logic in Whitehall, which does actually prevent extra translations on documents where the main language is not English. However you're right there is no reason to restrict it that way here, so changed to simply check the number of translations.